.pre-commit-config.yaml

```diff
@@ -19,15 +19,15 @@ repos:
         args: ['--line-length=120', '--include=\.pyi?$']
         exclude: gen/
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.3.0
+    rev: v1.4.1
     hooks:
       - id: mypy
         files: ^(src/ecalc/libraries/)
         exclude: tests|conftest.py
         args: ['--config-file=./pyproject.toml']
         additional_dependencies:
           - types-PyYAML
-          - pydantic
+          - pydantic==1.10.9
```

Comment:
it is only possible to specify a fixed version and not a range?

what about the other deps, set version for those as well?